docs/docs/data-structures/tree/schema-definition.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: Schema Definition
+sidebar_position: 3
+---
+
+import { PackageLink } from "@site/src/components/shortLinks";
+
+A `SchemaFactory` object is created by passing a unique string such as a UUID to the constructor.
```

Comment:
You should add some context here on what this doc is about. It starts talking about `schemaFactory` which seems kind of abrupt.